cdap-ui/app/cdap/components/KeyValuePairs/KeyValuePair.js

```diff
@@ -99,14 +100,14 @@ class KeyValuePair extends Component {
         <button
           type="submit"
           className="btn add-row-btn btn-link"
-          onClick={this.props.addRow}
+          onClick={(e) => {this.props.addRow(); preventPropagation(e);}}
```

Comment:
If my guess is correct, I think you don't need the `preventPropagation` if you remove the `type="submit"`. Not sure why we need the submit type

Reply:
Hmm removing the `type="submit"` still bubbles the click event for me